config/routes.rb

```diff
@@ -49,6 +49,9 @@
       collection do
         post :import
         resources :merge, only: [:new, :create], controller: 'issues/merge'
+        resources :states, only: [], controller: 'issues/states' do
+          collection { put :update_states }
+        end
```

Comment:
I think the controller name can be more descriptive, since we're just using it to update states. Maybe:

```suggestion
        resource :bulk_update_states, only: [:update]
```
Then we can PUT to `/projects/:project_id/issues/bulk_update_states`.

What do you think?